### PR TITLE
Added feature card and made recommended improvements

### PR DESCRIPTION
**Pull request by nfrenette**
Added feature card css and HTML example
Made improvements recommended in previous PR **https://github.com/wet-boew/GCWeb/pull/2156**

### CODE REVIEW

**Comment by nfrenette** [comment]
Resolved merged conflicts as discussed

File: méli-mélo/th-canadaday/canada-day.html [hunk 1]
```diff
@@ -186,26 +186,102 @@ <h3>CSS classes</h3>
...
 <pre><code>&lt;button class="btn btn-default bg-canadaday-beige" type="button"&gt;Button&lt;/button&gt;</code></pre>
-<h3>&lt;a&gt; .btn example</h3>
+<h4>&lt;a&gt; .btn example</h4>
 <a href="#" class="btn btn-default bg-canadaday-beige" role="button">Button</a>
-<h4>Code</h4>
+<h5>Code</h5>
 <pre><code>&lt;a href="#" class="btn btn-default bg-canadaday-beige" role="button"&gt;Button&lt;/a&gt;</code></pre>
+<h2 id="a3">Feature tiles using panels</h2>
+<h3>Appearance</h3>
+<div class="row wb-eqht mrgn-tp-md">
```
Comment (duboisp) [inline méli-mélo/th-canadaday/canada-day.html:214]: Suggestion: You can use the equal height CSS plugin which are more performant. 

```suggestion
<div class="row wb-eqht-grd mrgn-tp-md">
```

**Comment by duboisp** [comment]
@nfrenette for your information, see my suggestion ([last resolved conversation](https://github.com/wet-boew/GCWeb/pull/2162#pullrequestreview-1428373924)) to use the equal height pure CSS instead of the original plugin. `wb-eqht-grd`.

File: méli-mélo/th-canadaday/canada-day.html (hunk 1 above)
Comment (duboisp) [reply méli-mélo/th-canadaday/canada-day.html:214]: Resolving considering this is only a suggestion and optional.

**Pull request merged**